homeassistant/components/cloud/http_api.py

```diff
@@ -128,7 +128,8 @@ async def error_handler(
         try:
             result = await handler(view, request, *args, **kwargs)
             return result
-
+        except Unauthorized:
+            raise
```

Comment:
Please add tests for these lines or remove them as they are currently not used.

Keeping it has only the advantage, that the two decorators `@require_admin` and `@_handle_cloud_errors`
can be swapped without problems.

With the fact that the integration is tested well, we can remove the lines as CI and code review verifies that the two decorators are set in correct order.

Reply:
Right, we can just remove this one 👍

Reply:
Done in [0205380](https://github.com/home-assistant/core/pull/97804/commits/0205380e06756359cdcf44b0b08755fee828e7bb)